dev/sparktestsupport/modules.py

```diff
@@ -256,9 +256,21 @@ def __hash__(self):
 )
 
 
+mllib_local = Module(
+    name="mllib-local",
+    dependencies=[sketch],
```

Comment:
Sounds fair! Actually, I think it's better to just use `IllegalArgumentException` here.